tests/hpxml_translator_test.rb

```diff
@@ -1111,6 +1111,7 @@ def _test_multiple_hvac(xmls, hvac_multiple_dir, hvac_base_dir, all_results)
     puts "Multiple HVAC test results:"
     xmls.sort.each do |xml|
       next if not xml.include? hvac_multiple_dir
+      next if xml.include? "evap-cooler" # skip for evap cooler, neither energy nor capacity is linearly scaled for evap cooler
```

Comment:
Though good to check multiple and partial systems working as expected (with scaled airflow sizes, able to meet setpoint, etc.), there's no linearly scaled meter to check its output relationship.